test/e2e/storage/volume_expand.go

```diff
@@ -166,6 +163,38 @@ var _ = utils.SIGDescribe("Volume expand", func() {
 		pvcConditions := pvc.Status.Conditions
 		Expect(len(pvcConditions)).To(Equal(0), "pvc should not have conditions")
 	})
+
+	It("should allow expansion of block volumes", func() {
+		pvc, storageClassVar, err = setupFunc(true /*allowExpansion*/, true /*blockVolume*/)
+
+		By("Waiting for pvc to be in bound phase")
```

Comment:
fyi, when moving these tests to the framework, they will need to be able to handle delayed binding, so the pod needs to be created first.